docs/user-guide/08-compiling.md

```diff
@@ -179,6 +179,13 @@ we can compile the `computeMain()` entry point to SPIR-V using the following com
 slangc hello-world.slang -target spirv -o hello-world.spv
 ```
 
+> #### Note ####
+> The `slangc` CLI [does not currently support](https://github.com/shader-slang/slang/issues/5541) `[shader(...)]` attributes for targets other than SPIRV. For other targets, the `-entry` argument is mandatory. Example:
```

Comment:
I don't think it is accurate to say `slangc` doesn't support `[shader()]` attribute because shaders with `[shader()]` attribute will be correctly compiled by slangc and the attribute will be reflected in the resulting code.

It is just that for targets without multi-entrypoint support, `-stage` or `-profile` argument is still needed even if the shader contains `[shader()]` attribute when using `slangc`.

Reply:
Updated. Please suggest alternate wording if still preferred.

Reply:
I believe that OptiX also supports multiple entry points via the [shader()] attribute. Might be better to more specifically list the targets that are currently known to not support multiple entry points.

Reply:
How about 
> The `slangc` CLI currently cannot automatically deduce `-entrypoint` and `-stage`/`-profile` options from the `[shader(...)]` attributes when generating code for targets other than SPIRV, Metal, CUDA, or Optix. For targets such as HLSL, please continue to specify `-entry` and `-profile` options even when compiling a file with `[shader(...)]` attributes on the entrypoints.

Reply:
Updated with specified targets. Added hyperlinks.